package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@acidicMercury8 can we remove package files now?

Reply:
@PeyTy we previously had a problem with the inability to use the `setup-nodejs` action due to the absence of at least an empty manifest. I guess in the future we will be able to get rid of them after creating our Hexa action

Reply:
@PeyTy good, done. I guess it would be better also to add correct Hexa build to Teapot